ddtrace/internal/telemetry/writer.py

```diff
@@ -152,18 +152,19 @@ def add_event(self, payload, payload_type):
             self._events_queue.append(event)
 
     def enable(self):
-        # type: () -> None
+        # type: () -> bool
         """
         Enable the instrumentation telemetry collection service. If the service has already been
         activated before, this method does nothing. Use ``disable`` to turn off the telemetry collection service.
         """
         if self.status == ServiceStatus.RUNNING:
-            return
+            return True
 
         self._enabled = True
         self.start()
 
         atexit.register(self.stop)
+        return True
```

Comment:
```suggestion
       if config._telemetry_enabled:  # we need to defined this configuration
            self.start()

            atexit.register(self.stop)
            return True
        return False
```

Reply:
what is `_telemetry_enabled`? the `DD_INSTRUMENTATION_TELEMETRY_ENABLED` env var?
```
_telemetry_enabled = asbool(os.getenv("DD_INSTRUMENTATION_TELEMETRY_ENABLED", True))
```

Reply:
Yup sorry that was unclear. We can do this in a follow up or make the change here